system/modules/core/templates/jquery/j_mediaelement.html5

```diff
@@ -9,7 +9,12 @@ $GLOBALS['TL_CSS'][] = 'assets/jquery/mediaelement/'. $GLOBALS['TL_ASSETS']['MED
 <script>
   (function($) {
     $(document).ready(function() {
-      $('video,audio').mediaelementplayer({
+
+      var media_items = $('video').filter(function() {
+        return $(this).parent().prop("tagName") !== "PICTURE";
```

Comment:
And yes we could return the plain javascript element with .parent()[0] or .parent().get(0).

Looks like this: $(this).parent().get(0).tagName.toLowerCase() !== 'picture'

Or in vanilla js:

this.parentElement.tagName.toLowerCase();